weeder.cabal

```diff
@@ -20,6 +20,10 @@ extra-source-files:
     test/Spec/*.stdout
     test/Spec/*.failing
 
+source-repository head
+  type:     git
+  location: git://github.com/ocharles/weeder.git
```

Comment:
I generally prefer the https form as it's clickable form Hackage. WDYT?

Reply:
Yeah, makes sense. I just copy-pasted this from another repo without much thought.

Reply:
JSYK @georgefst the git protocol is actually no longer available on GitHub https://github.blog/security/application-security/improving-git-protocol-security-github/